wiki/Tournaments/CFC/2018/en.md

```diff
@@ -0,0 +1,235 @@
+   - [Goose house - Hikaru nara \[Sc4's Rain\]](https://osu.ppy.sh/beatmapsets/519023/#fruits/1113160)
+   - [Akiko Shikata - Umineko no Naku Koro ni \~Rengoku\~ \[AHO\]](https://osu.ppy.sh/beatmapsets/21079/#fruits/77996)
+   - [rerulili - Noushou Sakuretsu Girl \[dksslqj Style\]](https://osu.ppy.sh/beatmapsets/63434/#fruits/208935)
+- FreeMod
+   - [Hige Driver join. SELEN - Dadadadadadadadadada \[EX\]](https://osu.ppy.sh/beatmapsets/350076/#fruits/771879)
+   - [Helblinde - Grief & Malice \[Madness\]](https://osu.ppy.sh/beatmapsets/325817/#fruits/723610)
+   - [Sagara Kokoro - Hoshizora no Ima \[S.S\]](https://osu.ppy.sh/beatmapsets/79116/#fruits/221950)
+   - [Kitsune^2 - Rainbow Tylenol \[Hell\]](https://osu.ppy.sh/beatmapsets/20789/#fruits/72585)
+- Tiebreaker 
+  - [DystopiaGround - AugoEidEs \[Ressurection\]](https://osu.ppy.sh/beatmapsets/537671/#fruits/1139268)
+  
+### Quarterfinals
+
+**[Download the mappack here! (109.8MB)](https://mega.nz/#!0gdwkDLQ!UVRknCV5eRScWDLbHzGESuryQmeAHDOP1W7mnABIZ68)**
+
+- NoMod
+  - [PUSHER - Feel U \[Overdose\]](https://osu.ppy.sh/beatmapsets/740666/#fruits/1562529)
+  - [M2U - Marigold (feat. Guriri) \[Alice\]](https://osu.ppy.sh/beatmapsets/605710/#fruits/1279904)
+  - [Qrispy Joybox - licca \[Special\]](https://osu.ppy.sh/beatmapsets/242877/#fruits/560453)
+  - [Himeringo - Yotsuya-san ni Yoroshiku \[Winber1's Extreme\]](https://osu.ppy.sh/beatmapsets/100049/#fruits/378781)
+- Hidden
+   - [Hotaru Murasaki - Re:TrymenT \[Ascendance's DreaM\]](https://osu.ppy.sh/beatmapsets/518004/#fruits/1209998)
+   - [M2U - Quo Vadis \[Extreme\]](https://osu.ppy.sh/beatmapsets/171421/#fruits/485284)
+   - [Caravan Palace - Rock It For Me \[Insane\]](https://osu.ppy.sh/beatmapsets/69041/#fruits/212118)
+- Hard Rock
+   - [ClariS - Connect -TV MIX- \[Spec's Overdose\]](https://osu.ppy.sh/beatmapsets/203567/#fruits/500293)
+   - [Utagumi Setsugekka - Yumemi Sunrise \[Hard\]](https://osu.ppy.sh/beatmapsets/132312/#fruits/338841)
+   - [Kagamine Rin - Oni Kanojo \[gow's Insane\]](https://osu.ppy.sh/beatmapsets/23616/#fruits/86245)
+- Double Time
+   - [Mili - Utopiosphere \[Platter\]](https://osu.ppy.sh/beatmapsets/187177/#fruits/446767)
+   - [Function Phantom - Algebra \[Sinnoh's Platter\]](https://osu.ppy.sh/beatmapsets/468281/#fruits/1442004)
+   - [senya - Magica \[SatelliteFrostMix\]](https://osu.ppy.sh/beatmapsets/91791/#fruits/248474)
+- FreeMod
+   - [nano - Nevereverland \[Rain\]](https://osu.ppy.sh/beatmapsets/149570/#fruits/439454)
+   - [Roselia - LOUDER \[Crystal's Rain\]](https://osu.ppy.sh/beatmapsets/595580/#fruits/1270273)
+   - [Hyadain - Hyadain no Joujou Yuujou (TV Size) \[Saten's SELAMAT PAGI!\]](https://osu.ppy.sh/beatmapsets/32775/#fruits/112586)
+- Tiebreaker 
+  - [UNDERVEIL ALLSTARS - UNDERVEIL IS REAL!!! ...AND WE ARE UNDERVEIL!!! \[*Together as one, we stay determined!\]](https://osu.ppy.sh/beatmapsets/636920/#fruits/1351241)
```

Comment:
`\[*Together as one, we stay determined!\]` -> `\[\*Together as one, we stay determined!\]`